### PR TITLE
Update field descriptions for container cluster

### PR DESCRIPTION
**Pull request by trodge**
If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [x] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by zli82016** [comment]
Can you please update the documentation, too? https://github.com/GoogleCloudPlatform/magic-modules/blame/main/mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

**Comment by trodge** [comment]
> Can you please update the documentation, too? https://github.com/GoogleCloudPlatform/magic-modules/blame/main/mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

Done. It looks like the field descriptions were already updated in the docs.

File: mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown [hunk 1]
```diff
@@ -1177,11 +1177,11 @@ and all pods running on the nodes. Specified as a map from the key, such as
...
 
-* `workload_vulnerability_mode` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Sets which mode to use for Protect workload vulnerability scanning feature. Accepted values are WORKLOAD_VULNERABILITY_MODE_UNSPECIFIED, DISABLED, BASIC.
+* `workload_vulnerability_mode` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Sets which mode to use for Protect workload vulnerability scanning feature. Accepted values are DISABLED, BASIC.
 
 <a name="nested_workload_config"></a>The `protect_config.workload_config` block supports:
 
-* `auditMode` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) WorkloadConfig defines the flags to enable or disable the workload configurations for the cluster. Accepted values are MODE_UNSPECIFIED, DISABLED, BASIC.
+* `auditMode` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) WorkloadConfig defines the flags to enable or disable the workload configurations for the cluster. Accepted values are DISABLED, BASIC.
```
Comment (zli82016) [inline mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown:1184]: The description in the doc does not match with its description for the field.

Comment (zli82016) [reply mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown:1184]: The description should be `Sets which mode of auditing should be used for the cluster's workloads. Accepted values are DISABLED, BASIC.`

Comment (trodge) [reply mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown:1184]: Should be fixed now, thanks.

**Review by zli82016 — APPROVED** [review]
LGTM

**Pull request merged**